packages/headless/src/api/commerce/commerce-api-client.test.ts

```diff
@@ -190,6 +196,7 @@ describe('commerce api client', () => {
         language: request.language,
         currency: request.currency,
       },
+      requestMetadata: {method: 'search/querySuggest'},
```

Comment:
Shouldn't we also have an it for search/productSuggest?

Reply:
Done in https://github.com/coveo/ui-kit/pull/4086/commits/91b6b2b7aed99c128f87a54efb9333f197749002